src/components/app/Details.css

```diff
@@ -11,7 +11,7 @@
   align-items: center;
   justify-content: space-between; /* This pushes the sidebar button to the right */
   border: solid var(--grey-30);
-  border-width: 1px 0 1px;
+  border-width: 1px 0;
```

Comment:
This is [shorthand-property-no-redundant-values](https://stylelint.io/user-guide/rules/list/shorthand-property-no-redundant-values/)